docs/user/TerraformAzureAuthentication.md

```diff
@@ -18,8 +18,8 @@ To authenticate Terraform to access Azure, you will need to set the following fo
 | :--- | ---: | ---: | ---: |
 | tenant_id | your Azure tenant id | string  |
```

Comment:
The tenant_id and subscription_id items seem to be missing a column definition.

Reply:
It should be handled by the formatting row, but fixed it.